strr-platform-web/app/pages/platform/dashboard.vue

```diff
@@ -41,9 +41,12 @@ onMounted(async () => {
   } else {
     // existing registration or application under the account
     // set left side of header
-    title.value = platformBusiness.value?.legalName
+    title.value = platformBusiness.value.legalName
     subtitles.value = [
-      { text: platformBusiness.value?.homeJurisdiction || t('text.notAvailable') },
+      ...(platformBusiness.value.homeJurisdiction
+        ? [{ text: platformBusiness.value.homeJurisdiction }]
+        : []
```

Comment:
added this because I think they may want to make 'homeJurisdiction' optional eventually